### PR TITLE
Add requirements.txt

### PR DESCRIPTION
**Pull request by andrewda**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+opencv-python
+tqdm
+numpy
+argparse
+imgaug
+keras
+tensorflow
+h5py
```
Comment (SUPERCILEX) [inline requirements.txt]: New line plz?

**Comment by JacksonCoder** [comment]
@andrewda I didn't know which versions to use, but now that I have them installed, I can add another commit with the versions I'm using.

File: requirements.txt (hunk 1 above)
Comment (JacksonCoder) [reply requirements.txt]: requirements.txt files aren't necessarily a part of Python, but they are part of pip. (Python's package manager). With requirement files, txt is the standard format. Also, I can add a newline if you want.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+opencv-python==3.3.0.10
+tqdm==4.19.4
+numpy==1.13.3
+imgaug==0.2.5
+tensorflow==1.4.0
+h5py==2.7.1
+
```
Comment (andrewda) [inline requirements.txt]: Get rid of this extra newline then good to go

Comment (JacksonCoder) [reply requirements.txt]: Oh, sorry! Thought that @SUPERCILEX wanted that.

Comment (SUPERCILEX) [reply requirements.txt]: My B for not being clear: you should have 1 line at the end of a file, but not 2 (you have 2 here).

**Comment by andrewda** [comment]
K imma call this good to go, even though OpenCV won't install through pip in 90% of cases.

**Pull request merged**